app/views/api-docs/swagger/index.html.erb

```diff
@@ -1,24 +1,5 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <meta charset="utf-8" />
-  <meta name="ROBOTS" content="NOODP" />
-  <meta name="viewport" content="initial-scale=1" />
-  <title>e-Manifest API Documentation</title>
-  <%= stylesheet_link_tag 'swagger-ui' %>
-  <%= csrf_meta_tags %>
-</head>
-<body class="swagger-section">
-  <header>
-    <div class='container'>
-      <div>
-        <h1>e-Manifest</h1>
-        <%= link_to 'Home', root_path %>
-        <%= link_to 'Industry', new_submission_path %>
-      </div>
-    </div>
-  </header>
-<%= render "javascript" %>
-<%= render 'swagger_ui/swagger_ui', discovery_url: 'swagger.json' %>
-</body>
-</html>
+<%= stylesheet_link_tag 'swagger-ui/screen' %>
+<div class="swagger-section usa-width-one-whole">
+  <%= render "javascript" %>
```

Comment:
The reason that the original implementation did not use the application layout is that the `javascript` invocation in the main app layout happens at the end of the *body* tag, and the swagger_ui partial needs jquery to already be loaded. Basically loading the javascript before the body rather than after it.